docs/developers_guide/landice/api.rst

```diff
@@ -414,3 +419,6 @@ Landice Framework
    mesh.set_rectangular_geom_points_and_edges
    mesh.set_cell_width
    mesh.get_dist_to_edge_and_GL
```

Comment:
```suggestion
   mesh.get_dist_to_edge_and_gl
```

Reply:
resolved in [5543fa9](https://github.com/MPAS-Dev/compass/pull/590/commits/5543fa9aa0f2a1aef4d68f066d2683a9aba6b6a7)